schema/xml/metaschema-markup-multiline.xsd

```diff
@@ -92,4 +125,11 @@
       <xs:enumeration value="right"/>
     </xs:restriction>
   </xs:simpleType>
+
+  <xs:complexType name="blockQuoteType">
+    <!-- TODO: confirm if nested blockquote is supported in markdown -->
```

Comment:
Minor, non-blocking: do we mean Markdown generally or specifically Markdown? It seems the original description (I could jokingly call it a specification 😆) did support it, and [CommonMark supports it and tests for it](https://spec.commonmark.org/dingus/?text=%3E%20deeply%20nested%20blockquote%0A%3E%3E%20deeply%20nested%20blockquote%0A%3E%3E%3E%3E%20deeply%20nested%20blockquote%0A%3E%3E%3E%3E%3E%20deeply%20nested%20blockquote%0A%3E%3E%3E%3E%3E%3E%20deeply%20nested%20blockquote). Is this what you're looking for? (This example is from the commonmark.js test suite itself.)

Reply:
Indeed, it's a big help. :+1:

Indeed this is probably true of any child of HTML `blockquote` especially lists, for reasons not unrelated to list nesting.

If something is hard to parse and we mandate it, it is one thing if CommonMark already does it, and another if no one really does it, or no one does it testably and consistently. (Unfortunately, giving one simple example and saying 'we do this' doesn't actually cut it since it's no real help to show only the happy path.)

Reply:
Well I was asking because CommonMark is what we are aiming at conformance with (sorry I wrote Markdown twice 🤦), it seems we have confirmed that CommonMark allows nested blockquotes and quickly testing one of their reference implementations backs this, no?